docs/tutorials/build-files-archive-with-dal/using-full-slot.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Part 4: Using the entire slot"
```

Comment:
```suggestion
title: "Part 5: Using the entire slot"
```

Reply:
I've corrected the step numbering, thanks for pointing that out.